bluesky/preprocessors.py

```diff
@@ -974,6 +976,14 @@ def _normalize_devices(devices):
     return devices, coupled_parents
 
 
+def __get_result_of_message(msg_type: str, obj):
+    result = yield Msg(msg_type, obj)
+    if result is None:
+        # this plan may be being list-ified
+        print(f"*** all positions for {obj.name} are relative to current position ***")
```

Comment:
Though this was merely copied from the original code, I wonder if there's no better destination for such information. 

It'd be surprising to have this print be a side-effect of listifying a plan generator.

Reply:
Captured in #1552